ebuild-writing/misc-files/patches/text.xml

```diff
@@ -30,15 +30,17 @@ mirrored</uri>, preferably on the Gentoo Infrastructure. When
 mirroring patches, choosing a name that will not cause conflicts is
 important — the <c>${P}</c> prefix is highly recommended
 here. Mirrored patches are often compressed with <c>xz</c> or
-<c>bzip2</c>. Remember to list these patches in <c>SRC_URI</c>.
+<c>bzip2</c>. Remember to list these patches in <c>SRC_URI</c>. Please
+see <uri link="::ebuild-maintenance/new-ebuild/#The files Directory"/>
+for more information.
 </p>
 
 <note>
 Patches included in <c>${FILESDIR}</c> should never be compressed.
 </note>
 
 <warning>
-Starting from EAPI=6 a strip patch level was limited to the <c>-p1</c>.
+Starting from EAPI=6, the patch strip level was limited to the <c>-p1</c>.
```

Comment:
I'd say something like
> Starting from EAPI=6, the patch strip level defaults to `-p1`.

I feel like _limited_ sounds weird.

Reply:
+1
"to the -p1" sounds strange as well.

Reply:
Thanks for the suggestions.